src/pages/ThorChainLP/components/AddLiquidity/AddLiquidityInput.tsx

```diff
@@ -1515,116 +1502,111 @@ export const AddLiquidityInput: React.FC<AddLiquidityInputProps> = ({
     virtualRuneDepositAmountFiatUserCurrency,
   ])
 
+  const handleDepositSubmit = useCallback(() => {
+    return isUnsafeQuote ? setShouldShowWarningAcknowledgement(true) : handleSubmit()
+  }, [handleSubmit, isUnsafeQuote])
```

Comment:
Should this be handled by the acknolwedgement component itself?

Reply:
Actually disregard me - since the intent is to have this pop up on click, that's actually correct! `<AddLiquidityInput />` does things the right way and `<TradeInput />` is the odd one which needs to be implemented the same way i.e pop the ack CTA up on click